docs/getting-started/concepts.md

```diff
@@ -7,67 +7,67 @@ Iter8 is the Kubernetes release optimizer built for DevOps, MLOps, SRE and data
 
 ## Use-cases
 
-Iter8 simplifies a variety of traffic engineering and metrics-driven validation use-cases. They are illustrated below.
+Iter8 simplifies a variety of traffic engineering and metrics-driven validation use-cases. To support such use cases, Iter8 provides support for the key challenges enabling simpler implementation and quicker adoption.
 
-![Iter8 use-cases](images/iter8usecases.png)
+**Progressive release with automated traffic management** Iter8 supports blue-green and canary release of new application and ML models. When new models are deployed, Iter8 automatically reconfigures the routing to desired traffic pattern. Deployment of new versions and their promotion is done by describing the desired state.
 
-## Design
+**A/B/n testing with client SDK and business metrics** Iter8 addresses the challenge of doing A/B/n testing of backend application components/ML models. It provides a simple client SDK allowing a user-facing component to easily reliably associate business metrics with the backend components that are used. This SDK provides stick lookup based on user request headers.
```

Comment:
```suggestion
**A/B/n testing with client SDK and business metrics** Iter8 addresses the challenge of doing A/B/n testing of backend application components/ML models. It provides a simple client SDK allowing a user-facing component to easily and reliably associate business metrics with the backend components that were used. This SDK provides stick lookup based on user request headers.
```

> This SDK provides stick lookup based on user request headers.
I don't know what stick look up is. Unless you mean to say static?

Reply:
sticky